cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,15 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+                Example: `# Create a Cluster Workflow Template:
+argo cluster-template create FILE1
+	
+# Create a Cluster Workflow Template with relaxed validation:
+argo cluster-template create FILE1 --strict=false
```

Comment:
```suggestion
  argo cluster-template create FILE1 --strict false
```

- Let's keep the format of the flags the same throughout
- this conventionally is indented by 2 spaces